packages/web-components/src/components/va-minimal-header/va-minimal-header.scss

```diff
@@ -3,7 +3,7 @@
   margin-left: auto;
   margin-right: auto;
   max-width: 1000px;
-  background-color: var(--color-primary-darkest);
+  background-color: var(--color-primary-darker);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)